web/themes/custom/hoeringsportal/templates/content/node--hearing--teaser.html.twig

```diff
@@ -0,0 +1,59 @@
+{# Template for Hearing teaser extending base-card #}
+
+{% extends 'themes/custom/hoeringsportal/templates/components/base-card.html.twig' %}
+
+{# Set the content state #}
+{% set content_state = content.field_content_state['#items'].value %}
+
+{# Set how many replies there is to this hearing #}
+{% set repliesCount = hearing_helper.getNumberOfReplies(node) %}
+
+{# Set date for next meeting #}
+{% set startdate = content.field_start_date['0']['#attributes'].datetime %}
+
+{# Show a date splash on the card #}
+{% block card_date %}
+  {# If we have a date then show a date splash on the card #}
+  {% if (content.field_start_date) %}
+    {# Show the date #}
+      <div class="fw-semibold">{{ startdate|date('d')|t }}</div>
```

Comment:
Use `format_date` …

Reply:
I'm not sure how to use `format_date` when I  want to show the date and month separately?

Reply:
I have created new formats to use with `format_date`.